network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/formassembly_body.html

```diff
@@ -24,17 +24,25 @@
     - for postal code field: {% if show_postal_code_field %}  ... {% endif %}
     - for comment field:     {% if not show_comment_field == "none" %} ... {% endif %}
 
-6. Get localization working again.
+6. If using a required checkbox field, add the classname 'reqMark' to the label wrapping the 
+   span element with the classname of `input-checkbox-faux`. This will ensure that the 
+   "required" asterisk renders to the right of the checkbox text. 
+    
+7. Get localization working again.
     - Localize form fields. e.g., {% trans "text to be translated" %} or {% blocktrans %} ... {% endblocktrans %}
     - Localize "title" attribute and label text for the following fields:
         - first name, last name, email, country, postal code, comments, privacy notice, email subscription
     - Localize "value" attribute for <input type="submit" ...>
     - Go to formassembly_head.html and make sure everything has been updated according to the instructions
 
-7. Update the revision note on the next line. Revision number can be found on https://mozillafoundation.tfaforms.net/versions/index/9
+8. Update the revision note on the next line. Revision number can be found on https://mozillafoundation.tfaforms.net/versions/index/9
 
 The code snippet below is based on FormAssembly form revision #32
 
+*** NOTE *** 
+We have added a custom label that states "* indicates a required field" to the template. 
```

Comment:
Hi @mmmavis I was wondering if you thought it would make more sense to leave this comment here, or in `petition.html` where the label actually is being defined.

My thinking is that it makes more sense here since we are telling the user that they should take the label into account when making changes to the form/template, but was curious to hear what you think is best.

Thanks!

Reply:
Thanks for sharing your thoughts. Leaving the note here sounds good. For extra clarification, I'm wondering what you think about updating this line to

```suggestion
We have added a custom label that states "* indicates a required field" to the template (i.e., in `petition.html`). 
```

Reply:
That sounds great to me! Updated 👍